superset-frontend/src/common/components/Modal.tsx

```diff
@@ -81,6 +81,11 @@ const StyledModal = styled(BaseModal)`
       margin-left: 8px;
     }
   }
+
+  // styling for Tabs component
+  .ant-tabs {
+    margin-top: -18px;
```

Comment:
Not sure the context of this exact measurement, or if gridUnit makes sense to use here too.

Reply:
I changed it to `-gridUnit*4`, I guess 2px won't make a big difference